drivers/volume/kdmp/kdmp.go

```diff
@@ -371,6 +375,24 @@ func (k *kdmp) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.
 	}
 	return volumeInfos, nil
 }
+
+// Check whether to run kopia executor in debug mode
+func enableKopiaExecutorDebugMode() bool {
+	logrus.Info("enableKopiaExecutorDebugMode entering")
```

Comment:
This is ok while developing but not for production, suggestion to remove this info message.